web-components/src/components/phone-input/PhoneInput.ts

```diff
@@ -49,6 +51,21 @@ export namespace PhoneInput {
       });
```

Comment:
In order for this to not be a breaking change, we need another attribute setting for users to explicitly opt-in to a `flags` feature so that all current users who like it the way it is do not have to update their software.
it could be like 
`@property({ type: Boolean, attribute: "show-flags" }) showFlags = false;`